Makefile

```
@@ -4,6 +4,7 @@
 
 GO_CMD ?= go
 GO_FMT ?= gofmt
+GO_VERSION := $(shell awk '/go /{print $$2}' go.mod|grep -v v)
```

This is a bit hack'ish and breaks easily I think as it can return multiple lines of data. I'd suggest to harden it a bit by making sure that we only return the first matching line (we know the go version is specified in the beginning of the file) and also requiring that the string `go` is in the beginning of the line.

```suggestion
GO_VERSION := $(shell awk '/^go /{print $$2}' go.mod|head -n1)
```


 We could also get the version with 
```
go list -m -f '{{.GoVersion}}'
```

But I guess that's not so nice as it makes the build dependent on some (recent enough) golang version being installed on the host 🤔